docs/design-document.md

```diff
@@ -554,15 +526,19 @@ The workflow is as follows:
    workflow is performed for each Git reference in the new entries, relative to
    the local state of each reference. If verification fails, abort and warn
    user. Note that the verification workflow must fetch each Git reference to
-   its corresponding remote tracker, `refs/remotes/origin/<ref>`. TODO: discuss
-   if verification is skipped for entries that work with namespaces not present
-   locally.
+   its corresponding remote tracker, `refs/remotes/origin/<ref>`.
 1. For each modified Git reference, update the local state. As all the refs have
    been successfully verified, each ref's remote state can be applied to the
    local repository, so `refs/heads/<ref>` matches `refs/remotes/origin/<ref>`.
 1. Set local RSL to the remote RSL's tip.
 
-#### RSLPush: Submitting local RSL changes
+NOTE: Some aspects of this workflow are under discussion and are subject to
+change. The gittuf implementation does not implement precisely this workflow.
+Specifically, the implementation does not verify new entries in the remote
+automatically. Additionally, the RSL may contain entries for references a client
+does not have, making verification of those entries unfeasible.
```

Comment:
This seems weird to have right after you present an algorithm.   Why not have the current text here and open an issue with the discussion.   You can point to that here.

Reply:
https://github.com/gittuf/gittuf/issues/708

Reply:
Added inline.